html-tables/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Tables</title>
+  </head>
+  <body>
+    <h1>Student Grade Table</h1>
+    <table>
+      <thead>
+        <tr>
+          <td><strong>Student Name</strong></td>
```

Comment:
Rather than using a `td` here, you should use a `th`, which will also remove the need for the `strong` element